src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseChecklistServiceTests.cs

```diff
@@ -376,11 +398,14 @@ public async Task GetChecklist_FullyValid()
                     new Update
                     {
                         Reason = "Test reason 1",
-                        Release = originalRelease
+                        ReleaseId = releaseId,
+                        Created = DateTime.UtcNow.AddDays(-13),
```

Comment:
It's a bit confusing to follow what makes the dates valid and invalid. Maybe the reason could be "Update copied from original release" and Created could be `Created = originalRelease.Created`?

Reply:
I've added some comments as well and added a new Update to the originalRelease so that hopefully its easier to follow what's going on.

Reply:
I changed it again - decided to simplify and only have one Update on `release`.